### PR TITLE
DEFEAT RECAPTCHA

### PR DESCRIPTION
**Pull request by joeyagreco**
[IMPLEMENTED FIX FOUND HERE](https://github.com/SeanDaBlack/KelloggBot/issues/23#issuecomment-991616078)

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -66,6 +75,58 @@ def generate_account(driver):
     time.sleep(1.5)
     driver.find_element_by_xpath(ACCEPT_BUTTON).click()
     time.sleep(2)
+    googleClass = driver.find_elements_by_class_name(CAPTCHA_BOX)[0]
```
Comment (pws1453) [inline main.py:78]: I'm almost wondering if a different function should be implemented for handling captchas outside of generate_account. Maybe the same code in a handle_captcha function with param driver? It's nitpicky, and the PR should still be accepted if the change isn't implemented, but I think readability could improve a bit.

**Review by pws1453 — APPROVED** [review]
Besides the note in main.py about creating a separate function for captcha handling, code runs great. Your code readability and quality is excellent. As a university student, I've been taking notes about it. @SeanDaBlack, please merge into main whenever possible.

File: main.py [hunk 2]
```diff
@@ -155,6 +217,28 @@ def random_email(name=None):
...
+
+
+def saveFile(content, filename):
+    with open(filename, "wb") as handle:
+        for data in content.iter_content():
+            handle.write(data)
+
+
 def main():
     while True:
```
Comment (pws1453) [inline main.py:243]: If desired, the multiprocessing implementation from https://github.com/SeanDaBlack/KelloggBot/pull/21 could be placed here.

**Review by jmrushing — APPROVED** [review]
I tested your additions on my fork, works great.
I do agree with other suggestions and PRs that we need multi-threading, a resume pdf generator, more code cleanup (what's been done looks great so far though), also need targeted phone number generation (these wacky area codes and extensions are easily filtered out), BUT these are all general gripes and not your fault - this PR should be accepted IMO :grin:

**Comment by SeanDaBlack** [comment]
This does the same as #28

**Pull request closed without merging**